tests/em/tdem/test_TDEM_DerivAdjoint_galvanic.py

```diff
@@ -7,7 +7,7 @@
 
 plotIt = False
 
-testDeriv = True
+testDeriv = False
```

Comment:
why this is changed to False?

Reply:
ah, thanks for catching this - it shouldn't have been (I must have forgotten about it when running some tests). I will update